### PR TITLE
0.20.0 Release

### PR DESCRIPTION
**Pull request by MCeley**
### Changed

- Upgraded the Android library to use Heap version 1.10.+

### Fixed

- Fixed type definition for `withReactNavigationAutotrack` on React Navigation 6.

## Checklist
- [x] Detox tests pass
- [x] If this is a bugfix/feature, the changelog has been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,16 +9,21 @@ __BEGIN_UNRELEASED__
...
-
-- Fixed type definition for `withReactNavigationAutotrack` on React Navigation 6.
-
 ### Security
 __END_UNRELEASED__
 
+## [0.20.0] - 2022-04-28
+
+### Changed
+- Upgraded the Android library to use Heap version 1.10.+
```
Comment (bnickel) [inline CHANGELOG.md:21]:
```suggestion
- Upgraded the Android library to use Heap version 1.10.+.
```

File: integration-tests/drivers/TestDriver066/package-lock.json [hunk 2]
```diff
@@ -2119,9 +2119,9 @@
       }
     },
     "node_modules/@heap/react-native-heap": {
-      "version": "0.19.0",
-      "resolved": "file:../../heap-react-native-heap-0.19.0.tgz",
-      "integrity": "sha512-GTeA2RuqmBEuCbX31NbGLiAEo3f7hj0ZjFfxWjiqkmKKyYS4pXENYhd0uQlxV+1K+PlymhR95tvE4t0L1jQJ3g==",
+      "version": "0.20.0",
```
Comment (bnickel) [inline integration-tests/drivers/TestDriver066/package-lock.json:2122]: This is a much more reasonable change size now that we're not reformatting this whole file. 😄

**Review by bnickel — APPROVED** [review]
LGTM!

**Review by apyousef — APPROVED** [review]
@MCeley LGTM

**Pull request merged**